docs/validators.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Validation functions
+author: Roxane Letourneau
+---
+
+Taquito provides functions that allow us to see if an address, a chain, a key hash, a contract address, a public key or a signature is valid. The `ValidationResult` returned by these functions is an enum which can take the following values:
```

Comment:
It would be worth explaining that the validation is done based on checksums to see if it is valid, and that it does not rely on a node, or tell the user if the address/key hash/block hash actually exists on a chain.